### PR TITLE
Add first ADRs (Using Prettier + Using Ohm)

### PR DESCRIPTION
**Pull request by charlespwd**

### CODE REVIEW

File: docs/decisions/ADR_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Why the decision was made
+- Alternatives considered
+-->
+
+## Consequences
+
+<!--
+Discuss the consequences or drawbacks of the decided approach.
+If your answer is "there are none," it's possible you haven't
+fully considered the impact of your decision.
```
Comment (karreiro) [inline docs/decisions/ADR_TEMPLATE.md:30]: @charlespwd What do you think about including a comment here suggesting to discuss possible "rollback"/"evolution" approaches?

That would be nice to document solutions for future known problems and also give a hint of how to possibly "undo" the decision or part of it.

**Review by karreiro — APPROVED** [review]
Thank you, @charlespwd! Awesome stuff 🚀 I've left only one minor/optional comment :)

File: docs/decisions/ADR_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Decision record template
+
+<!-- Inspired by: https://github.com/joelparkerhenderson/architecture-decision-record/blob/main/templates/decision-record-template-by-michael-nygard/index.md -->
+<!-- Start by updating "Title", and delete everything before it. -->
+
+# Title
+
+## Status
+
+<!-- What is the status? If it was decided, when did that happen? -->
```
Comment (mgmanzella) [inline docs/decisions/ADR_TEMPLATE.md:10]: It might be worth providing suggested statuses

**Review by mgmanzella — APPROVED** [review]
so stoked to see ADRs!! 🔥

**Pull request merged**